cli/CommandExecutor.cpp

```diff
@@ -117,10 +115,11 @@ void executeDescribeTable(
     // Printed column needs to be wide enough to print:
     //   1. The attribute name (in the printed "header").
     //   2. Any value of the attribute's Type.
-    int attr_column_width = attr.getDisplayName().length();
-    int type_column_width = attr.getType().getName().length();
-    max_attr_column_width = std::max(max_attr_column_width, attr_column_width);
-    max_type_column_width = std::max(max_type_column_width, type_column_width);
+    max_attr_column_width =
+        std::max(max_attr_column_width,
+            static_cast<int>(attr.getDisplayName().length()));
+    max_type_column_width = std::max(max_type_column_width,
+                                static_cast<int>(attr.getType().getName().length()));
```

Comment:
To wrap this line, there are a few ways, but unfortunately, not using the current one. On the other hand, the way you did for `max_attr_column_width` is ok. You may want to do the same thing to `max_type_column_width`.

Basically, the goal of a line wrapping is for readability so that we could distinguish it with another one-line statement. The main idea to wrap a line is to add four whitespace indentation counting from the first letter of the previous line. If the wrapped line is a part of a function call involving other variables (in this case, two arguments in `std::max`), we typically align the variable with other variables (See my code piece below).

```
  max_type_column_width = std::max(max_type_column_width,
                                   static_cast<int>(attr.getType().getName().length()));
```

This following way is also acceptable, but it is not as good as the previous one, since it does not present in a clear way that the expression in the wrapped line has a strong relationship with the `max_type_column_width` above.
```
  max_type_column_width = std::max(max_type_column_width,
      static_cast<int>(attr.getType().getName().length()));
```